content/docs/getting-started/connect-dbaas.md

```diff
@@ -64,3 +65,23 @@ Configure `cURL` with DBaaS credentials.
 ```sh
 alias curl="curl -s -k -u \"${NUODB_CP_USER}:${NUODB_CP_PASSWORD}\""
 ```
+
+### Setting up the Terraform Provider

```

Comment:
Is this subsection providing context for the `terraform` examples that show below? Is there anything we want to say here about where the configuration snippets should be inserted, e.g. into the same configuration that has the provider?